src/Sentry/Internal/Hub.cs

```diff
@@ -24,14 +24,17 @@ public Hub(SentryOptions options)
             Debug.Assert(options != null);
             _options = options;
 
-            if (options.Dsn == null)
+            if (string.IsNullOrWhiteSpace(options.Dsn))
```

Comment:
In Sentry SDKs, `""` means: Disable SDK.

If dsn was null here, we try to lookup.
If dsn was `""`, we should just let the program run, the SDK will be disabled (don't try to lookup for a DSN in env var, don't throw exception, etc)

```suggestion
            if (options.Dsn is null)
```

Reply:
Ah I see.  Will change.
Interestingly, no tests had failed ;)